README.md

```diff
@@ -1,41 +1,79 @@
+([McGillicuddy et al. 2009](http://www.ncbi.nlm.nih.gov/pubmed/19573811)).
+Therefore, detecting inactivation solely by sequencing the NF1 gene can result
+in false negatives. Because we have previously identified compounds that are
+synthetically lethal in NF1 inactivated cells
+([Wood et al. 2011](http://www.ncbi.nlm.nih.gov/pubmed/21697395)),
+the ability to detect patients with NF1 inactivation signatures could inform
+treatment decisions.
+
+## Code
+
+* A step-by-step guide of all analyses for the NF1 inactivation project is
+provided in `ANALYSIS.sh`
+* All required R packages are installed by `INSTALL.R`.
+* `scripts/` - all the main functions and pipeline scripts are provided
+  * |- `util/` - several helper functions including:
+    * |- `cancer_cv.py` - cross validation logic to ensure equal partitioning
+    * |- `get_y_class.py` - determine mutation status for genes in `X` matrix
+    * |- `transform.r` - transforming RNAseq distributed data into microarray
+  * |- `viz/` - using ggplot to vizualize classifier performance
+
+## Contact
 
-###################
-# CONTACT
-###################
-Please report all bugs and direct analysis questions to: `gregway@mail.med.upenn.edu`
+Please report all bugs and direct analysis questions to:
+`gregway@mail.med.upenn.edu`
 
-Please direct all other correspondence to: `csgreene@mail.med.upenn.edu` or `yolanda.sanchez@dartmouth.edu`
+Please direct all other correspondence to: `csgreene@mail.med.upenn.edu` or
+`yolanda.sanchez@dartmouth.edu`
 
-###################
-# ANALYSIS
-###################
-To reproduce and build upon our analyses, clone the repository and run `./ANALYSIS.sh`.
+## Analysis
+
+To reproduce and build upon our analyses, clone the repository and run
+`./ANALYSIS.sh`.
 
 RECOMMENDED:
-To ensure reproducibility our analyses should be performed using the provided [Docker image](https://hub.docker.com/r/gregway/nf1_inactivation).
-
-###################
-# DATA
-###################
-All data is publicly available. TCGA data used to train the classifier was retrieved from [UCSC Xena](https://genome-cancer.soe.ucsc.edu/proj/site/xena/datapages/). All of our validation data was deposited in [GEO](http://www.ncbi.nlm.nih.gov/geo/).
-
-###################
-# DEPENDENCIES
-###################
-All analyses are performed in Anaconda Python 3.5.1 (see `environment.yml`). Visualizations are built using R version 3.2.3. For specific package dependencies see `Docker/Dockerfile`
-
-###################
-# ACKNOWLEDGEMENTS
-###################
-This work was supported by the Genomics and Computational Biology graduate group at The Unversity of Pennsylvania (G.P.W); the Gordon and Betty Moore Foundation’s Data-Driven Discovery Initiative (grant number GBMF 4552 to C.S.G.); and the American Cancer Society (grant number IRG 8200327 to C.S.G.).
+To ensure reproducibility our analyses should be performed using the provided
```

Comment:
Should the link to docker hub (and possible this whole sentence) be moved to "Dependencies" with the other docker info?